core/variant/variant_utility.cpp

```diff
@@ -866,6 +866,13 @@ String VariantUtilityFunctions::error_string(Error error) {
 	return String(error_names[error]);
 }
 
+String VariantUtilityFunctions::type_string(Variant::Type p_type) {
+#ifdef DEBUG_ENABLED
+	ERR_FAIL_INDEX_V_MSG((int)p_type, (int)Variant::VARIANT_MAX, "<invalid type>", "Invalid type argument to type_string(), use the TYPE_* constants.");
+#endif
```

Comment:
I would keep this in release builds, we don't usually exclude such simple checks from release, unless on a very hot path (which doesn't seem to be the case here).

Reply:
> I would keep this in release builds, we don't usually exclude such simple checks from release, unless on a very hot path (which doesn't seem to be the case here).

Do you mean just getting rid of the `#ifdef` like this?

```suggestion
ERR_FAIL_INDEX_V_MSG((int)p_type, (int)Variant::VARIANT_MAX, "<invalid type>", "Invalid type argument to type_string(), use the TYPE_* constants.");
```